tests/static-check/run_checks.sh

```diff
@@ -43,6 +52,13 @@ cd "$(dirname $0)"/../../
 
 failure=0
 failures=""
+
+# in jenkins the workdir is already autogen'd
+# in github it is not, so do that work here
+if [ ! -f configure ]; then
+  ./autogen.sh --enable-debug
```

Comment:
This runs `./configure` as well. We don't need that, AFAIK, so it should be prefixed with `NO_CONFIGURE=1`.

Reply:
right.